### PR TITLE
project2-admission-trial

### PR DESCRIPTION
**Pull request by balqees0325**
A feature branch was created.
included in the feature branch are 
- linters file
-  index.html file
- style.css file
- readme.md file

### CODE REVIEW

**Review by smcommits — COMMENTED** [review]
Hi @balqees0325, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not contain any review. Please follow the instructions from [this guide](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/review_your_project.md) and provide a review on your project. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by balqees0325 — COMMENTED** [review]
That was an awesome job, well done.
However, in your subsequent project, try to use more comments in your codes to be easy for anyone to walk through.
cheers

File: README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+## Author
+
+👤 **Balqees Ibrahim**
+
+- GitHub: [@balqees0325](https://github.com/balqees0325)
+- Twitter: [@qeescode](https://twitter.com/qeescode)
+- LinkedIn: [Balqees Ibrahim](https://www.linkedin.com/in/balqees-ibrahim-911a81204/)
+
+## Live Demo
```
Comment (balqees0325) [inline README.md:37]: Well done for a good job.
However the live link on the readme file is not working, try to check that.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+					<li><a href="#global">Global variables</a></li>
+					<li><a href="#constant">Constants</a></li>
+					<li><a href="#data">Data types</a></li>
+					<li><a href="#condition">if...else statement</a></li>
+					<li><a href="#function">Function declarations</a></li>
+					<li><a href="#refrence">Refrence</a></li>
+				</ul>
+			</div>
+			<!-- end of side navigation -->
+			<div class="main">
```
Comment (balqees0325) [inline index.html:30]: On line 30 of index.html file. Try to use semantics HTML so as to make the website more visible to search engines.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+				<div class="intro">
```
Comment (balqees0325) [inline index.html:31]: Also at line 31 index.html Try to use semantics HTML , using a  <section> in place of <div>
You are doing a good job,just try to improve and you will be better soon.

File: .github/workflows/linters.yml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
```
Comment (usmansbk) [inline .github/workflows/linters.yml:46]: This is an example of Code review.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+					</small>
+					<p>
+						Select the code in the pad and hit Ctrl+R to watch it unfold in your
+						browser!
+					</p>
+				</div>
+			</div>
+		</div>
+	</body>
+</html>
```
Comment (balqees0325) [inline index.html:130]: An overall a good job, nice code very clear, 
In subsequent projects try to add comments to your codes to help you or any other person walk through your code.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
+# Admission trial project
+
+> Set up a Project2-trial folder.
+
+![image](https://user-images.githubusercontent.com/66781780/124811245-4eba5700-df73-11eb-8a75-484d4cce4dcc.png)
+
+In this project, we will set up a "A copy-admission-trial-project" repository. No complex coding is required for this exercise. Our goal here is to master all of the tools and best practices we learned about in previous steps.
+
+## Built With
```
Comment (usmansbk) [inline README.md:9]: You add comments to the changes. You basically say what you think could be improved.

**Review by usmansbk — COMMENTED** [review]
# Sample Code Review
This is an example of a review. See the comments below

**Review by balqees0325 — COMMENTED** [review]
The overall was a good project, good job at your end, However I will like to encourage you to add good comment to your codes to make it easy for anyone to pick up.
Thank you.

File: index.html (hunk 5 above)
Comment (usmansbk) [reply index.html:130]: I can see you've already reviewed your code here. Good job!

**Pull request merged**